### PR TITLE
Add device to Fanvil V serie 

### PR DESCRIPTION
**Pull request by fouille**
* V62
* V64
* V65
* update V67

### CODE REVIEW

File: plugins/wazo-fanvil/common/common.py [hunk 1]
```diff
@@ -200,6 +204,7 @@ def configure_common(self, raw_config):
             tpl = self._tpl_helper.get_template(f'common/{tpl_filename}')
             dst = os.path.join(self._tftpboot_dir, filename)
             raw_config['XX_fw_filename'] = fw_filename
+            raw_config['XX_model_info'] = _
```
Comment (bloom1) [inline plugins/wazo-fanvil/common/common.py:207]: Why `_`?

File: plugins/wazo-fanvil/serie-v/plugin-info [hunk 2]
```diff
@@ -1,12 +1,42 @@
 {
-    "version": "1.0.1",
+    "version": "1.0.3",
+    "description": "Plugin for Fanvil V6x series",
+    "description_fr": "Greffon pour La serie V6x Fanvil",
+    "version": "1.0.2",
```
Comment (bloom1) [inline plugins/wazo-fanvil/serie-v/plugin-info:5]: Duplicate keys

**Review by bloom1 — CHANGES_REQUESTED** [review]

File: plugins/wazo-fanvil/common/common.py (hunk 1 above)
Comment (fouille) [reply plugins/wazo-fanvil/common/common.py:207]: to add phone set model dynamically in plugins `/wazo-fanvil/common/templates/common/model-v6x.tpl` (for exemple) firmware download info file

Comment (bloom1) [reply plugins/wazo-fanvil/common/common.py:207]: This is not clear nor good practice. Please define explicitly what the value should be. I guess you want it to be  `fw_filename`. In that case write `fw_filename`. `_` is just an ugly hack that Python provides that should be used when you want to discard a value or use the previous value in an *interactive session* (when you get a REPL). This is not an interactive session.
Thank you!

Comment (fouille) [reply plugins/wazo-fanvil/common/common.py]: @bloom1 fix done

**Review by bloom1 — APPROVED** [review]

**Comment by tyor123-pixel** [comment]
Hello everyone, please Do someone has fanvil 67 template for 3cx ? I cannot find anywhere

**Pull request merged**